docs/backlog-management.md

```diff
@@ -0,0 +1,55 @@
+# Backlog management
+All FFC teams follow an agile delivery approach.  Teams should use whichever agile methodology best fits their delivery, for example Scrum, Kanban etc.
+
+If teams are unclear which methodology to adopt, they should start with vanilla Scrum and iterate as appropriate.  Teams looking for help getting started with Scrum can follow this [training series](http://scrumtrainingseries.com).
```

Comment:
Might be worth adding a link to the GDS Agile delivery section of the service manual for teams that are new to agile.
https://www.gov.uk/service-manual/agile-delivery

Reply:
Good idea

Reply:
Link to guidance added.

Reply:
Looks good to me.